Source/WebKit/Shared/unix/AuxiliaryProcessMain.cpp

```diff
@@ -34,10 +34,6 @@
 #include <string.h>
 #include <wtf/text/StringToIntegerConversion.h>
 
-#if USE(LIBWPE) && !ENABLE(BUBBLEWRAP_SANDBOX)
-#include "ProcessProviderLibWPE.h"
-#endif
```

Comment:
The header really is used, though.

Reply:
Yes, it's coming from somewhere else, or the build would fail.

Removing it wasn't a good choice because it makes future build failure more likely. Shrug. :)